.dockerignore

```diff
@@ -0,0 +1,42 @@
+.git
+.gitignore
+README.md
+
+#
+# OS X
+#
+.DS_Store
+.AppleDouble
+.LSOverride
+
+#
+# Rails
+#
+.env
+.env.sample
+*.rbc
+capybara-*.html
+.rspec
```

Comment:
This may pose a problem as @dleve123 pointed out in https://github.com/artsy/artsy-hokusai-templates/pull/6

Reply:
Thanks! Fixed in 40cac1c